libbeat/monitoring/report/log/log.go

```diff
@@ -43,15 +44,13 @@ var gauges = map[string]bool{
 	"filebeat.harvester.open_files":  true,
 	"beat.memstats.memory_total":     true,
 	"beat.memstats.memory_alloc":     true,
+	"beat.memstats.rss":              true,
 	"beat.memstats.gc_next":          true,
 	"beat.info.uptime.ms":            true,
 	"beat.cpu.user.ticks":            true,
-	"beat.cpu.user.time":             true,
```

Comment:
These metrics don't exist so I moved them. There are `beat.cpu.*.time.ms` metrics, but I don't see a problem with those being reported as delta (I assume the cpu times only ever increase).